src/Microsoft.VisualBasic/tests/ConversionsTests.cs

```diff
@@ -5,83 +5,3201 @@
+
+            // int.
+            yield return new object[] { 0, byte.MinValue };
+            yield return new object[] { 1, (byte)1 };
+            yield return new object[] { 0, byte.MinValue };
+            yield return new object[] { IntEnum.Value, (byte)1 };
+
+            // ulong.
+            yield return new object[] { ulong.MinValue, byte.MinValue };
+            yield return new object[] { (ulong)1, (byte)1 };
+            yield return new object[] { ulong.MinValue, byte.MinValue };
+            yield return new object[] { ULongEnum.Value, (byte)1 };
+
+            // long.
+            yield return new object[] { 0, byte.MinValue };
+            yield return new object[] { 1, (byte)1 };
+            yield return new object[] { 0, byte.MinValue };
+            yield return new object[] { LongEnum.Value, (byte)1 };
+
+            // float.
+            yield return new object[] { (float)0, byte.MinValue };
+            yield return new object[] { (float)1, (byte)1 };
+
+            // double.
+            yield return new object[] { (double)0, byte.MinValue };
+            yield return new object[] { (double)1, (byte)1 };
+
+            // decimal.
+            yield return new object[] { (decimal)0, byte.MinValue };
+            yield return new object[] { (decimal)1, (byte)1 };
+
+            // bool.
+            yield return new object[] { true, byte.MaxValue };
+            yield return new object[] { false, byte.MinValue };
+
+            // null.
+            yield return new object[] { null, byte.MinValue };
+
+            // string.
+            yield return new object[] { "0", byte.MinValue };
+            yield return new object[] { "1", (byte)1 };
+            yield return new object[] { "&h5", (byte)5 };
+            yield return new object[] { "&h0", byte.MinValue };
+            yield return new object[] { "&o5", (byte)5 };
+            yield return new object[] { " &o5", (byte)5 };
+            yield return new object[] { "&o0", byte.MinValue };
+            yield return new object[] { 1.1.ToString(), (byte)1 };
+        }
+
+        [Theory]
+        [MemberData(nameof(ToByte_Object_TestData))]
+        public void ToByte_Object_ReturnsExpected(object value, byte expected)
+        {
+            AssertEqual(expected, Conversions.ToByte(value));
+        }
+
+        public static IEnumerable<object[]> ToByte_InvalidObject_TestData()
+        {
+            yield return new object[] { char.MinValue };
+            yield return new object[] { (char)1 };
+            yield return new object[] { char.MaxValue };
+            yield return new object[] { new DateTime(10) };
+            yield return new object[] { "" };
+            yield return new object[] { "&" };
+            yield return new object[] { "&a" };
+            yield return new object[] { "&a0" };
+            yield return new object[] { "true" };
+            yield return new object[] { "false" };
+            yield return new object[] { "invalid" };
```

Comment:
add ```MemberData(nameof(ToByte_InvalidString_TestData)``` to the test instead of duplicating them here

Reply:
Converted to `InvalidString_TestData`